desk/sur/forums.hoon

```diff
@@ -239,142 +245,142 @@
+            ^-  value
+            ?>  ?=(%s -.child-ids)
+            [%s (~(del in p.child-ids) post-id.act)]
+        ==  ==
+      =?  database.rock  !=(parent-id.act-post 0)
+        =<  +
+        %+  ~(q db database.rock)  %forums
+        :*  %update  thread-table  [%s %post-id [%& %eq parent-id.act-post]]
+        :~  :-  %reply-ids
+            |=  reply-ids=value
+            ^-  value
+            ?>  ?=(%s -.reply-ids)
+            [%s (~(del in p.reply-ids) post-id.act)]
+        ==  ==
+      ::
+      ::  remove rows for this post in all relevant databases (post, thread)
+      ::
+      ::
       =.  database.rock
         =<  +
         %+  ~(q db database.rock)
-          %forums  
-        :*  %delete  thread-table
-            :+  %and 
-              [%s %thread-id [%& %eq id.act]] 
-            [%s %author [%& %eq src.bowl]]
-        ==
+          %forums
+        [%delete thread-table %s %post-id %& %eq post-id.act]
       =.  database.rock
         =<  +
         %+  ~(q db database.rock)
-          %forums  
-        :*  %delete  post-table
-            :+  %and 
-              [%s %post-id [%& %eq id.act]] 
-            [%s %author [%& %eq src.bowl]]
-        ==
+          %forums
+        [%delete post-table %s %post-id %& %eq post-id.act]
       rock
     ::
         %edit-content
-      ::  TODO: 
+      ::  TODO:
       ::  1. Scry groups to obtain permissions
       ::  2. Check if src.bowl is author OR has the appropriate permissions
       ::  Look at steps outlined in %delete-post for guidance...
-      =/  author=@p  author:(get-post database.rock post-table post-id.act)
-      ?>  =(author src.bowl)
+      =/  act-post=post  (get-post database.rock post-table post-id.act)
+      =/  [[@da author=@p @t] edits]  (pop:om-hist p.history.act-post)
+      ?.  =(author src.bowl)
+        ~|('%forums: edit permissions denied' !!)
       :-  metadata.rock
       =<  +
       %+  ~(q db database.rock)
-        %forums  
+        %forums
       :*  %update  post-table  [%s %post-id [%& %eq post-id.act]]
-          :~  :-  %content
-              |=  content=value
-              ^-  value
-              content.act
-            ::
-              :-  %editors
-              |=  editors=value
-              ^-  value
-              ?>  ?=([%s *] editors)
-              ::  Only add to editors if not the author
-              ?:  =(author src.bowl)
-                editors
-              [%s (~(put in p.editors) src.bowl)]
-            ::
-              :-  %history
-              |=  history=value
-              ^-  value
-              ?>  ?=([%b *] history)
-              =+  ;;  ((mop @da ,[who=@p content=@t]) gth)  p.history
-              [%b (put:om-hist - now.bowl [src.bowl content.act])]
+      :~  :-  %history
+          |=  history=value
+          ^-  value
+          ?>  ?=([%b *] history)
+          =+  ;;(edits p.history)
+          [%b (put:om-hist - now.bowl [src.bowl content.act])]
       ==  ==
```

Comment:
same suggestion as in the other two spots above. nest the colsig inside of the coltar and make sure the luslus are aligned with their initiating runes.

Reply:
Fixed, and I think this answers my question above (assuming that "luslus" refers to the "tistis" delimiters for the `:*` and `:~` expressions)